lib/controller.js

```diff
@@ -557,6 +557,28 @@ controller.provisionTessel = function(opts) {
     });
 };
 
+
+controller.restoreTessel = function(opts) {
+  if (!opts.openwrt) {
+    return Promise.reject('You must use the --openwrt option to restore.');
+  }
+
+  opts.usb = true;
+  opts.lan = false;
+  opts.altSetting = 1;
+  return controller.get(opts).then(function(tessel) {
+    return new Promise(function(resolve, reject) {
```

Comment:
Can this be changed to just return `tessel.restore(opts)` since that is a Promise?